lib/matplotlib/fontconfig_pattern.py

```diff
@@ -177,19 +177,34 @@ def _property(self, s, loc, tokens):
 parse_fontconfig_pattern = lru_cache()(FontconfigPatternParser().parse)
 
 
+def _escape_val(val, escape_func):
+    if type(val) == list:
+        val = [escape_func(r'\\\1', str(x)) for x in val
+               if x is not None]
+        if val != []:
```

Comment:
Also a bit unusual. `if val:` would do.

Anyway, if we have an empty list, that would get returned. But I assume we always want a string. So actually an unconditional `val = ','.join(val)` is the way to go (evaluates to an empty string for an empty list).